### PR TITLE
[ToDo] Add custom emojis for todo

### PR DESCRIPTION
**Pull request by Just-Jojo**
## Type

- [ ] Bugfix
- [ ] Enhancement
- [x] New Feature
- [ ] Other

## Description of the changes
Allows users to set custom emojis for their todo lists, providing that the bot can use them

### CODE REVIEW

File: todo/commands/settings.py [hunk 1]
```diff
@@ -225,6 +235,57 @@ async def todo_autosort(self, ctx: commands.Context, value: bool):
...
+        if reset:
+            await self.cache.set_user_setting(ctx.author, "todo_emoji", None)
+            return await ctx.send("Done. Your emoji has been reset.")
+        msg = await ctx.send("Please react with the emoji you want for the uncompleted todos")
+        pred = ReactionPredicate.same_context(message=msg, user=ctx.author)
+        try:
+            emoji = (await self.bot.wait_for("reaction_add", check=pred))[0].emoji
+        except asyncio.TimeoutError:
+            return await ctx.send("Okay, I won't set your emojis")
+        if not isinstance(emoji, str) and not self.bot.get_emoji(emoji.id):
```
Comment (Kreusada) [inline todo/commands/settings.py:258]: Unless I'm missing something here, how can a string have an `id` attribute?

Same applies for L284

File: todo/utils/formatting.py [hunk 2]
```diff
@@ -2,13 +2,14 @@
 # Licensed under MIT
 
 from typing import List
+import logging
 
 __all__ = ["_format_todos", "_format_completed", "_build_underline"]
 
+log = logging.getLogger("red.jojocogs.todo.formatting")
```
Comment (Kreusada) [inline todo/utils/formatting.py:9]:
```suggestion

__all__ = ["_format_todos", "_format_completed", "_build_underline"]

```
This variable isn't used here.

**Review by Kreusada — CHANGES_REQUESTED** [review]
Looks good, just 2 small comments I have.

File: todo/commands/settings.py (hunk 1 above)
Comment (Just-Jojo) [reply todo/commands/settings.py:258]: First `not` check guards against the emoji being a string, not the entire `if` statement

**Comment by Just-Jojo** [comment]
The requested changes have been made

**Review by Kreusada — APPROVED** [review]
Looks good to me

**Pull request merged**